2-mandatory/test.js

```diff
@@ -0,0 +1,11 @@
+PreviousPassword = [
```

Comment:
I always create a file called test.js to try all my code too!!! 😅
We both must have great minds! 👩‍🔬

Reply:
Yes It's much easier to manage the code